src/pages/index.js

```diff
@@ -10,13 +11,7 @@ import SiteTitle from '../components/SiteTitle';
 import css from './index.module.less';
 
 // images
-import custom from './images/enact-home-custom.svg';
-import auto from './images/enact-home-auto.svg';
-import easy from './images/enact-home-easy.svg';
-import hero from './images/enact-home-hero.svg';
-import perf from './images/enact-home-perf.svg';
 import starsSmall from './images/stars-small.svg';
-import ufo from './images/ufo.svg';
```

Comment:
Note: I deleted this local variables for images, because <StaticImage /> accepts only a string for `src` prop.
reference: https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-plugin-image/#restrictions-on-using-staticimage